python/run-tests.py

```diff
@@ -176,10 +177,24 @@ def run_delta_connect_codegen_python(root_dir):
 if __name__ == "__main__":
     print("##### Running python tests #####")
     root_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
-    package = prepare(root_dir)
+    prepare(root_dir)
 
     run_python_style_checks(root_dir)
     run_mypy_tests(root_dir)
     run_pypi_packaging_tests(root_dir)
     run_delta_connect_codegen_python(root_dir)
-    test(root_dir, package)
+    run_delta_connect_tests = os.getenv("RUN_DELTA_CONNECT_TESTS")
+    if run_delta_connect_tests is None or run_delta_connect_tests == "false":
+        delta_spark_package = get_local_package("delta-spark")
+        test(root_dir, "delta", [delta_spark_package])
+    elif run_delta_connect_tests == "true":
+        # TODO: In the future, find a way to get these
+        # packages locally instead of downloading from Maven.
+        delta_connect_packages = ["com.google.protobuf:protobuf-java:3.25.1",
+                                  "org.apache.spark:spark-connect_2.13:4.0.0-preview1",
+                                  get_local_package("delta-connect-server")]
```

Comment:
Why do we need the server package here?

Reply:
@allisonport-db We need the server code to test the client locally just like the Scala Client one.